tests/Doctrine/Tests/DBAL/Platforms/AbstractMySQLPlatformTestCase.php

```diff
@@ -207,6 +207,8 @@ public function testGetDateTimeTypeDeclarationSql() : void
 
     /**
      * {@inheritDoc}
+     *
+     * @psalm-return array
```

Comment:
Not sure what this means.

Reply:
The parent has `@psalm-return never-return`, hence this comment.